### PR TITLE
Xenial64

### LINKED ISSUE #43: Update vagrant file with new Ubuntu version
Update the vagrant script to pick https://app.vagrantup.com/ubuntu/boxes/xenial64 as documented here https://github.com/CCPPETMR/CCPPETMR_VM/pull/40

### PR DESCRIPTION
**Pull request by paskino**
New VM with ubuntu/xenial64. 

Closes #40.
Closes #43

Currently #44 is still open

### CODE REVIEW

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -63,15 +75,37 @@ Vagrant.configure("2") do |config|
...
+	apt-get install -y wget git xorg xterm gdm menu gksu synaptic gnome-session gnome-panel metacity --no-install-recommends
     apt-get install -y at-spi2-core gnome-terminal gnome-control-center nautilus 
     service gdm start
-
-    # To hide vagrant from login screen:
+    
+	# set the current locale, otherwise the gnome-terminal doesn't start
+	sudo locale-gen en_GB.UTF-8
+	sudo update-locale LANG=en_GB.UTF-8
+	sudo localectl set-x11-keymap us
```
Comment (KrisThielemans) [inline vagrant/Vagrantfile]: are you sure about using a us keyboard? Might be why I had to install a UK one. probably a good idea to add a few locales/keyboard maps from the beginning (like you did with portuguese at the time) and just tell people how to switch.

Comment (paskino) [reply vagrant/Vagrantfile]: Which locale would you want on board?

I though US keymap would be the most common. 
Which one should I use?

**Comment by KrisThielemans** [comment]
If it doesn't harm, I'd add quite a few locales really. just makes it easier. As we're a UK project, I do think we need a UK keyboard (which you've already done by now).

**Comment by paskino** [comment]
#44 is closed now.
https://github.com/CCPPETMR/CCPPETMR_VM/commit/8889fd903853a70e9f33a1d58143db429d2bb170 adds new locales.

VM builds correctly from Vagrantfile.

**Comment by paskino** [comment]
Shall I merge?

**Comment by KrisThielemans** [comment]
If you're happy, I'm happy. Yes

________________________________

**Comment by paskino** [comment]
changed base box to ubuntu xenial64
a bug with `grep -c` was fixed using `gawk`

**Pull request merged**